CLI/createUserCLI.js

```diff
@@ -1,7 +1,16 @@
 require('dotenv').config({
     path: '../.env'
 })
-const questions = require('./questions')
+const {
+    questions
+} = require('./questions')
+const {
+    verification_questions
+} = require('./questions')
+const {
+    password_questions
+} = require('./questions')
+
```

Comment:
For now, it's fine, but in general you should be able to put all of them together

```suggestion
const {
    questions, verification_questions, password_questions
} = require('./questions')

```